### PR TITLE
fixed gtk bug in outline pad

### PR DESCRIPTION
**Pull request by nosami**

### CODE REVIEW

File: MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs [hunk 1]
```diff
@@ -49,19 +52,28 @@ type FSharpOutlineTextEditorExtension() as x =
...
       if not refreshingOutline then
         refreshingOutline <- true
-        GLib.Timeout.Add (1000u, (fun _ -> refillTree())) |> ignore
+        timerId <- GLib.Timeout.Add (1000u, (fun _ -> refillTree()))
 
     override x.Initialize() =
         base.Initialize()
-        x.DocumentContext.DocumentParsed.Add(updateDocumentOutline)
+        handler <- x.DocumentContext.DocumentParsed.Subscribe(fun o e -> x.updateDocumentOutline())
+        ()
```
Comment (7sharp9) [inline MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs]: Whats this for?

Comment (nosami) [inline MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs]: Not sure this had any bearing on the fix. Wanted to dispose the handler.

File: MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs [hunk 2]
```diff
@@ -120,4 +132,8 @@ type FSharpOutlineTextEditorExtension() as x =
         member x.ReleaseOutlineWidget() =
             treeView |> Option.iter(fun tv -> Option.tryCast<ScrolledWindow>(tv.Parent) 
                                               |> Option.iter (fun sw -> sw.Destroy())
-                                              tv.Destroy())
+                                              let treeStore = tv.Model :?> TreeStore
```
Comment (7sharp9) [inline MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs]: use `match tv.Model with :? TreeStore as ts -> ...`

File: MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs (hunk 1 above)
Comment (7sharp9) [inline MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs]: I meant the dangling unit

Comment (nosami) [inline MonoDevelop.FSharpBinding/FSharpOutlineTextEditorExtension.fs]: ah yeah.. good catch

**Comment by 7sharp9** [comment]
Thanks!

**Pull request merged**